src/game/input.c

```diff
@@ -17,9 +17,32 @@ static bool m_KeyConflict[INPUT_ROLE_NUMBER_OF] = { false };
 static bool m_BtnConflict[INPUT_ROLE_NUMBER_OF] = { false };
 static int32_t m_HoldBack = 0;
 static int32_t m_HoldForward = 0;
+static int32_t m_HoldLook = 0;
 
+static void Input_CheckChangeTarget(INPUT_STATE *input);
 static INPUT_STATE Input_GetDebounced(INPUT_STATE input);
 
+static void Input_CheckChangeTarget(INPUT_STATE *input)
+{
+    if (g_Lara.gun_status != LGS_READY) {
+        return;
+    }
+
+    if (input->look) {
+        if (m_HoldLook >= 6)
+            m_HoldLook = 100;
+        else {
+            input->look = 0;
+            m_HoldLook++;
+        }
+    } else {
+        if (m_HoldLook && m_HoldLook != 100) {
```

Comment:
Can we remove the magic constants by doing away without `100` and putting `6` in a `#define`?
Eg
```c
#define LOOK_HOLD_TIME 6
if (input->look) {
    if (m_HoldLook < LOOK_HOLD_TIME) {
        input->look = 0;
        m_HoldLook++;
    }
} else {
    if (m_HoldLook > 0 && m_HoldLook < LOOK_HOLD_TIME) {
        input->change_target = 1;
    }
    m_HoldLook = 0;
}